### DISCUSSION TITLE
Budibase capabilities?

### DISCUSSION THREAD

**Discussion by danielhalper** [Budibase/budibase#4942] [General]
Hi, I am brand new to this and am unclear about the platform. Would a product manager (with only basic html, css, and SQL skills) be able to create an MVP on Budibase and then pass off the code base to their dev team? Can that project be exported as a well organized/structured Github project for the team to start working on seamlessly? 

As Product Designer/manager with below par dev skills, I'd love a no-code/low-code platform to create MVPs that can be carried on by a dev team that can continue developing it on their usual platforms. Please let me know!

**Comment by mjashanks** [comment]
A Product manager with these skills should be able to create some apps with Budibase. Sometimes it helps to be able to write short snippets of Javascript.

In terms of exporting the code - this is not possible. Budibase apps always run inside Budibase. If you are handing over the app, then the next person would need to continue using Budibase.

**Reply by danielhalper** [reply]
Thanks for the info! If that were a feature one day (even if it costs a big fee for the user to export), I would love to use this platform! It would be game changing for a product designer/manager to be able to create the MVPs without having to convince the dev team to move to a different platform.
